src/components/NcDateTimePickerNative/NcDateTimePickerNative.vue

```diff
@@ -0,0 +1,320 @@
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
+
+<docs>
+
+### General description
+
+This components provides a wrapper around the native browser datetime picker. <br>
+This is an input with some type of date e.g. https://html.spec.whatwg.org/multipage/input.html#local-date-and-time-state-(type=datetime-local). <br>
+All available types are: 'date', 'datetime-local', 'month', 'time' and 'week', please check them here: https://html.spec.whatwg.org/multipage/input.html#  <br>
+
+### Examples
+
+#### Usage: type='datetime-local'
+```vue
+<template>
+	<span>
+		<NcDateTimePickerNative
+			v-model="value"
+			:id="id"
+			:label="label"
+			type="datetime-local" />
+	</span>
+</template>
+<script>
+	export default {
+		data() {
+			return {
+				value: new Date(),
+				id: 'date-time-picker_id',
+				label: 'please select a new date',
+			}
+		},
+	}
+</script>
+```
+
+#### Usage: type='datetime-local' with min date and max date
+```vue
+<template>
+	<span>
+		<NcDateTimePickerNative
+			v-model="value"
+			:id="id"
+			:min="yesterdayDate"
+			:max="someDate"
+			:label="label"
+			type="datetime-local" />
+	</span>
+</template>
+
+<script>
+	export default {
+		data() {
+			return {
+				value: new Date(),
+				id: 'date-time-picker_id',
+				label: 'please select a new date',
+				yesterdayDate: new Date(new Date().setDate(new Date().getDate() - 1)),
+				someDate: new Date(new Date().setDate(new Date().getDate() + 7)),
+			}
+		},
+	}
+</script>
+```
+
+#### Usage: type='week'
+```vue
+<template>
+	<span>
+		<NcDateTimePickerNative
+			v-model="value"
+			:id="id"
+			:label="label"
+			type="week" />
+	</span>
+</template>
+
+<script>
+	export default {
+		data() {
+			return {
+				value: new Date(),
+				id: 'date-time-picker_id',
+				label: 'please select a new date',
+			}
+		},
+	}
+</script>
+```
+
+#### Usage: type='month'
+```vue
+<template>
+	<span>
+		<NcDateTimePickerNative
+			v-model="value"
+			:id="id"
+			:label="label"
+			type="month" />
+	</span>
+</template>
+
+<script>
+	export default {
+		data() {
+			return {
+				value: new Date(),
+				id: 'date-time-picker_id',
+				label: 'please select a new date',
+			}
+		},
+	}
+</script>
+```
+
+</docs>
+
+<template>
+	<div class="native-datetime-picker">
+		<label :class="{ 'hidden-visually': hideLabel }" :for="`id_${id}`">{{ label }}</label>
+		<input :id="`id_${id}`"
```

Comment:
is the hard coded `id_` suffix needed? Feels a bit weird to ask for an ID and then not using the provided value only.